bin/automatic_review.py

```diff
@@ -0,0 +1,250 @@
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from analysis_driver import rest_communication
+from analysis_driver.constants import ELEMENT_REVIEW_COMMENTS
+
+
+
+class RunReview():
+    def __init__(self, run_name):
+        self.run_name = run_name
+        self.run = self.read_run()
+        self.lane = self.read_lanes()
+
+    def read_run(self):
+        return rest_communication.get_documents(
+            'aggregate/all_runs',
+            match={"run_id":self.run_name}
+        )
+
+    def read_lanes(self):
+        return rest_communication.get_documents(
+            'aggregate/run_elements',
+            match={"run_id":self.run_name}
+        )
+
+    def run_metrics(self):
```

Comment:
This will be filtered by finished/not finished elsewhere when getting all runs to be reviews - remove